salt/metalk8s/addons/logging/fluent-bit/deployed/configmap.sls

```diff
@@ -68,40 +68,66 @@ Create fluent-bit ConfigMap:
                 Remove         BOOT_ID
                 Remove         UID
                 Remove         GID
+                Add            unit unknown
+            # Lift kubernetes labels as we have no "easy way" to get a specific
+            # nested field (especially if this one is not always available)
+            # Sees: https://github.com/fluent/fluent-bit/issues/2152
+            [FILTER]
+                Name           nest
+                Match          kube.*
+                Operation      lift
+                Nested_under   kubernetes
+                Add_prefix     kubernetes_
+            [FILTER]
+                Name           nest
+                Match          kube.*
+                Operation      lift
+                Nested_under   kubernetes_labels
+                Add_prefix     kubernetes_labels_
             [FILTER]
                 Name           modify
                 Match          kube.*
                 Remove         logtag
+                # Rename just skipped if dest key already exists
+                Rename         kubernetes_labels_app.kubernetes.io/name app
+                Rename         kubernetes_labels_app app
+                Rename         kubernetes_labels_release release
+                Rename         kubernetes_container_name container
+                Rename         kubernetes_host node
+                Rename         kubernetes_namespace_name namespace
+                Rename         kubernetes_pod_name pod
+                Remove_wildcard kubernetes_
```

Comment:
Shouldn't this be: `Remove_wildcard kubernetes_*`?

Reply:
No it's not needed